src/dolphin/util.ts

```diff
@@ -1,11 +1,13 @@
 import { settingsManager } from "@settings/settingsManager";
+import { app } from "electron";
 import log from "electron-log";
 import * as fs from "fs-extra";
-import * as ini from "ini";
```

Comment:
So if we previously used the `ini` package to parse the ini file, and now we're manually handling the ini parsing,  does that mean we don't need the `ini` package anymore?

Reply:
yep, i'll remove it.